lib/vtls/schannel.c

```diff
@@ -557,6 +557,20 @@ schannel_connect_step1(struct Curl_easy *data, struct connectdata *conn,
                    "names in server certificates.\n"));
     }
 
+    /* security request flags */
+    BACKEND->req_flags = ISC_REQ_SEQUENCE_DETECT | ISC_REQ_REPLAY_DETECT |
+      ISC_REQ_CONFIDENTIALITY | ISC_REQ_ALLOCATE_MEMORY |
+      ISC_REQ_STREAM;
+
+    if(!SSL_SET_OPTION(auto_creds)) {
+      schannel_cred.dwFlags &= ~SCH_CRED_USE_DEFAULT_CREDS;
+      schannel_cred.dwFlags |= SCH_CRED_NO_DEFAULT_CREDS;
```

Comment:
I don't understand why you seem to be clearing the bit before setting it, if there's a good reason for that it might justify a comment.

Reply:
I did this for posterity. It's not the same bit/flag. The flags have opposite meaning.

Reply:
Ah, skimmed the names too quick, sorry.